app/javascript/packs/load-service-panel.js

```diff
@@ -0,0 +1,35 @@
+const updateLeadService = (contact_id, lead_service_id, lead_service_note) => {
+  $.ajax({
```

Comment:
as I was saying before I don't think we need all this javascript, we can try to use the built in `form remote:true` options rails 5 has, and just re-render the partial, do you think you can try that way?